src/components/builder/prefill/prefill.stories.tsx

```diff
@@ -60,19 +60,25 @@ export const PluginBlankNoAttributes: Story = {
 
   play: async ({canvasElement}) => {
     const canvas = within(canvasElement);
+    const pluginInput = canvas.getByLabelText('Plugin');
+
+    await waitFor(
+      async () => {
+        await userEvent.keyboard('[Esc]');
+        pluginInput.focus();
+        await userEvent.keyboard('[ArrowDown]');
+        expect(canvas.queryByText('Plugin 1')).toBeInTheDocument();
+        expect(canvas.queryByText('Plugin 2')).toBeInTheDocument();
+      },
+      {timeout: 1500, container: canvasElement}
```

Comment:
I hope we will be able to get rid of this arbitrary timeout when moved to jest

Reply:
yeah I hope so to, the delays in the getter functions don't seem to reliably work and I have no idea what is or isn't happening. I'm hopeful the jest tests will be better with the mocked timers, which should also result in a faster test suite.